apps/aehttp/src/aehttpc_aeternity.erl

```diff
@@ -163,19 +199,45 @@ post_pin_tx(SignedSpendTx, NodeSpec) ->
     %lager:debug("PINNING: wrote to PC tx hash: ~p", [TxHash]),
     encode_child_pin_payload(TxHash).
 
+pin_contract_call(ContractPubkey, PinTx, Who, Amount, _Fee, SignModule) ->
+    Nonce = get_local_nonce(Who),
+    {ok, CallData} = aeb_fate_abi:create_calldata("pin", [{bytes, PinTx}]),
+    ABI = 3, % not really nice, what is the supported version of getting the latest ABI version
+    TxSpec =
+        #{  caller_id   => aeser_id:create(account, Who)
+          , nonce       => Nonce
+          , contract_id => aeser_id:create(contract, ContractPubkey)
+          , abi_version => ABI
+          , fee         => 1000000 * min_gas_price()
+          , amount      => Amount
+          , gas         => 1000000
+          , gas_price   => min_gas_price()
+          , call_data   => CallData},
+    {ok, Tx} = aect_call_tx:new(TxSpec),
+    NetworkId = aec_governance:get_network_id(),
+    SignedCallTx = sign_tx(Tx, NetworkId, Who, SignModule),
+    aec_tx_pool:push(SignedCallTx, tx_received).
+
+min_gas_price() ->
+    Protocol = aec_hard_forks:protocol_effective_at_height(1),
+    max(aec_governance:minimum_gas_price(Protocol),
+        aec_tx_pool:minimum_miner_gas_price()).
+
 get_pin_by_tx_hash(TxHashEnc, NodeSpec) ->
     case decode_child_pin_payload(TxHashEnc) of
-         error -> {error, {bad_child_pin_tx_hash, TxHashEnc}};
-         TxHash ->
+         {error, _} -> {error, {bad_child_pin_tx_hash, TxHashEnc}};
+         {ok, TxHash} ->
             TxPath = <<"/v3/transactions/", TxHash/binary>>,
             case get_request(TxPath, NodeSpec, 5000) of
-                {ok, #{<<"tx">> := #{<<"payload">> := EncPin}}} ->
+                {ok, #{<<"tx">> := #{<<"payload">> := EncPin}, <<"block_height">> := Height}} = Tx ->
+                    lager:debug("TXXXX: ~p", [Tx]),
```

Comment:
remove left over debug log

Reply:
fixed

Reply:
not really fixed, the lager debug should just have been removed, but fine for now